src/components/Header.astro

```diff
@@ -0,0 +1,62 @@
+---
+import { LanguageSelector } from 'astro-i18next/components';
+import { t } from 'i18next';
+
+const navigation = [
+  {
+    name: t('layout.header.home'),
+    href: '/',
+  },
+  {
+    name: 'Skill 08',
+    href: '#',
+  },
+  {
+    name: 'Skill 09',
+    href: '#',
+  },
+];
+---
+
+<header class="bg-indigo-600">
+  <nav class="mx-auto max-w-7xl px-6 lg:px-8" aria-label="Top">
+    <div
+      class="flex w-full items-center justify-between border-b border-indigo-500 py-6 lg:border-none"
+    >
+      <div class="flex items-center">
+        <a href="#">
+          <img
+            class="h-10 w-auto"
+            src="https://tailwindui.com/img/logos/mark.svg?color=white"
```

Comment:
our icon instead of tailwind one

Reply:
Unfortunately, I don't have any proper assets. I moved this to #12 so that we can finish this PR.